Add UCF 0.84

cnkeats opened the pull request:
Updates UCF from 0.80 to 0.84 via an external binary provided by Altimor

Companion PR to https://github.com/project-slippi/slippi-ssbm-asm/pull/122

cnkeats committed:
add UCF 0.84

NikhilNarayana committed:
add menu items

NikhilNarayana committed:
use ubuntu-latest for builds

NikhilNarayana committed:
chore(ci): update action steps

NikhilNarayana committed:
chore: shift ucf description 1 character

NikhilNarayana committed:
chore: 0.84 Stealth -> Stealth 0.84

NikhilNarayana committed:
chore: update UCF descrip

NikhilNarayana committed:
chore: update gecko bins

NikhilNarayana committed:
add note about description limits

NikhilNarayana committed:
chore: update 0.84 descrip

JLaferri commented on common/config/MeleeCodes.c:
```diff
@@ -50,23 +52,45 @@ const MeleeCodeOption cfOptionStealthUcf = {
+	"     Will enable UCF 0.8. Includes",
+	"     dashback, shield drop, and",
+	"     wiggle out of tumble",
 	"",
```
We should probably remove the extra spaces that were added here right? I'm pretty sure all the other code descriptions use two spaces to indent and now all the descriptions for controller fix use 3.

NikhilNarayana replied:
added a comment about why the extra spacing exists

NikhilNarayana committed:
add comment about spacing

JLaferri committed:
update codesets

https://github.com/project-slippi/slippi-ssbm-asm/pull/122/commits/f26aabc785107133273116f0572f237d60132acd
